auth-web/src/services/business.services.ts

```diff
@@ -93,4 +93,37 @@ export default class BusinessService {
       `${ConfigHelper.getLegalAPIV2Url()}/businesses/${businessIdentifier}/documents/summary`, config)
     CommonUtils.fileDownload(data?.data, `${businessIdentifier} Summary.pdf`, data?.headers['content-type'])
   }
+
+  static async fetchContinuationBusinesses (): Promise<any> {
```

Comment:
You are not fetching businesses. You are fetching (continuation) reviews.

Chat with Paul and Ketaki about what they're going to name the endpoints (re: line 125) and name this method accordingly.

Reply:
Just left this issue open: I will update the endpoint name when Paul gets back to office.

Reply:
OK. Maybe you can update this when you work on bcgov/entity#21068.